README.md

```diff
@@ -1,17 +1,26 @@
-# spring-cloud-azure
+# Spring Cloud Azure
 
-This project pulls together the efforts being worked on in conjunction with Microsoft
-to integrate Spring applications with Azure Cloud Services.  It hosts the gh-pages branch
-that provides the content for https://cloud.spring.io/spring-cloud-azure/.
+The project provides Spring Cloud integration with Azure services. Developers can use Spring programming model to unlock the power of Azure. It takes care of configurations and offers key features to interact with Azure services, such as Azure Storage and Azure Event Hub. 
 
-This project will not host source code directly, but provides a convenient jumping off
-point into the Microsoft repositories that provide code that support Spring applications:
+## Feature List 
 
-- https://github.com/Microsoft/azure-spring-boot
-  Starters/Autoconfiguration for active directory, document db, keyvault, media services, service bus, storage.
+Currently, Spring Cloud Azure offers the following features with samples.  
 
-- https://github.com/Microsoft/spring-data-documentdb
-  Spring Data DocumentDB support.
+- [Spring Resource Abstraction for Azure Storage](spring-cloud-azure-samples/spring-cloud-azure-storage-sample/README.adoc) 
```

Comment:
The displayed name is module name, but the link is sample. How about having separate section to put sample link? We should have a readme for each module to explain module details.